WcaOnRails/app/webpacker/javascript/edit-schedule/SchedulesEditor/EditScheduleForRoom.js

```diff
@@ -134,6 +140,14 @@ export default class EditScheduleForRoom extends React.Component {
           actionDetails={actionDetails}
         />
       </Row>
+    ) : (
+      <Alert bsStyle="warning">
+        Please add a timezone to the venue `&apos;`
+        {' '}
+        {venueWcif.name}
+        {' '}
+        `&apos;`
```

Comment:
In cases like this it's neat to use JS template strings, like this:

```jsx
{`Please add a timezone to the venue '${venueWcif.name}'`}
```

Reply:
When taking a look at the changes locally I noticed the message displayed was:
```
Please add a timezone to the venue `'` My Venue `'`
```
I think Jonatan suggestion is good: in my opinion it looks better without the space around the name, and it would also make sure only one quote is displayed.